src/pages/namju/Login/Login.js

```diff
@@ -1,8 +1,87 @@
-import React from 'react';
-import './Login';
+import React, { useState, useEffect } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import './Login.scss';
 
-const Login = () => {
-  return <div></div>;
-};
+function Login() {
+  const [inputValid, setInputValid] = useState(false);
+  const [inputValues, setInputValues] = useState({
+    userName: '',
+    password: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValues({
+      ...inputValues,
+      [name]: value,
+    });
+  };
+
+  useEffect(() => {
+    setInputValid(
+      inputValues.userName.includes('@') && inputValues.password.length >= 5
+    );
+  }, [inputValues]);
+
+  const navigate = useNavigate();
+  const goToMain = () => {
+    fetch('http://10.58.2.159/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: inputValues.userName,
+        password: inputValues.password,
+        // account: inputValues.userName,
+        // name: inputValues.userName,
+        // phone: '010-2937-1623',
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'USER_DOES_NOT_EXIST') {
+          alert('잘못입력했으니 확인 부탁');
+        } else if (result.message === 'INVALID_PASSWORD') {
+          alert('비번 확인해라!!!');
+        } else {
+          alert('냠냠굿~');
+          navigate('/main-namju');
+        }
+      });
+  };
+
+  return (
+    <div className="login-namju-viewport">
+      <div className="login-namju">
+        <h1 className="westagram-logo">Westagram</h1>
+        <form method="post" className="login-form" onSubmit={goToMain}>
+          <input
+            id="inputAccount"
+            name="userName"
+            type="text"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+            onChange={handleInput}
+            onKeyPress={e => (e.key === 'Enter' ? goToMain() : null)}
```

Comment:
삼항연산자에서 false일 때의 값이 null이면, 이 경우 && 단축평가로 사용할 수 있음
```suggestion
            onKeyPress={e => (e.key === 'Enter' && goToMain())}
```

Reply:
하지만 기능을 명확히 하기 위해선 함수를 선언하고 호출하는 게 나아 보임